Source/Private/ASDisplayNode+AsyncDisplay.mm

```diff
@@ -291,16 +299,23 @@ - (void)__willDisplayNodeContentWithRenderingContext:(CGContextRef)context drawP
       ASDisplayNodeContextModifier willDisplayNodeContentWithRenderingContext = _willDisplayNodeContentWithRenderingContext;
     __instanceLock__.unlock();
 
+    CGPathRef cornerRadiusPath = NULL;
     if (cornerRoundingType == ASCornerRoundingTypePrecomposited && cornerRadius > 0.0) {
       ASDisplayNodeAssert(context == UIGraphicsGetCurrentContext(), @"context is expected to be pushed on UIGraphics stack %@", self);
       // TODO: This clip path should be removed if we are rasterizing.
       CGRect boundingBox = CGContextGetClipBoundingBox(context);
-      [[UIBezierPath bezierPathWithRoundedRect:boundingBox cornerRadius:cornerRadius] addClip];
+      cornerRadiusPath = ASCGRoundedPathCreate(boundingBox, cornerRadius);
+      CGContextAddPath(context, cornerRadiusPath);
+      CGContextClip(context);
     }
     
     if (willDisplayNodeContentWithRenderingContext) {
       willDisplayNodeContentWithRenderingContext(context, drawParameters);
     }
+      
+    if (cornerRadiusPath) {
+      CGPathRelease(cornerRadiusPath);
+    }
```

Comment:
Any reason we shouldn't move this local variable inside the if-statement and release it unconditionally?

Reply:
Nope already did changed that in my update I will push in a minute :)

Reply:
Please push the change :P